VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -70,111 +70,124 @@ import NotificationsSettingsScreen from './ProfileScreen/SettingsScreen/Notifica
 const { WEBVIEW_URL_FACILITY_LOCATOR, LINK_URL_ABOUT_PACT_ACT } = getEnv()
 
 type HomeScreenProps = StackScreenProps<HomeStackParamList, 'Home'>
+const MemoizedLoadingComponent = React.memo(LoadingComponent)
```

Comment:
React Query re-renders the parent component whenever the state of a query is updated. Even though the re-renders are limited to the query fields that are referenced in the component, we're referencing quite a bit of fields from multiple queries, which causes multiple re-renders of the parent component, which also causes many child components like `LoadingComponent` to re-render as well. Whenever the `LoadingComponent` is re-rendered, the `Lottie` animation to reset its position, making the component look like it's glitching.

My initial approach was to try and use the `notifyOnChangeProps` option in the queries to limit re-renders. This option prevents the component from re-rendering until any of the listed properties change, e.g. `notifyOnChangeProps['isFetched']` will only re-render the component when the `isFetched` property is changed for that query. This wouldn't work for the Home screen's use case, because even if the Home screen only re-renders when the `isFetched` fields are updated, it would still re-render the animation after each query is finished fetching. 

The best way I could come up with for preventing re-renders was using `React.memo` to prevent the `LoadingComponent` from re-rendering if any of its props haven't changed.